perfkitbenchmarker/linux_benchmarks/cloudharmony_network_benchmark.py

```diff
@@ -306,6 +315,15 @@ def _Run(benchmark_spec, test):
     http_url = f'http://{benchmark_spec.bucket}.storage.googleapis.com/probe'
     endpoints = f'--test_endpoint={http_url}'
     _AddStorageMetadata(client, metadata)
+  elif FLAGS.ch_network_test_service_type == DNS:
+    vms = vm_groups['server']
+    # use GCP zonal internal DNS, but maybe should add domain to vm's data attributes?
```

Comment:
Hmm. yeah we probably want to set up a Domain. I don't know if GCP zonal internal DNS is what cloud harmony is interested in. @jellyfishcake  - Thoughts?